tests/Feature/BaseServiceTest.php

```diff
@@ -42,10 +42,10 @@ function it_enables_services()
             $mock->shouldReceive('imageIsDownloaded')->andReturn(true);
 
             // This is the actual assertion
-            $mock->shouldReceive('bootContainer')->with(['getmeili/meilisearch']);
+            $mock->shouldReceive('bootContainer')->once();
```

Comment:
Why this replacement?

Reply:
Without specifying a call count the default is zero or more - http://docs.mockery.io/en/latest/reference/expectations.html#declaring-call-count-expectations

Tests were currently passing no matter the value of `bootContainer` or `getmeili/meilisearch`

Reply:
HA! Thanks @morrislaptop! Could you *keep* the original `with`?

Reply:
Hi @mattstauffer the original with isn't correct, it's going to have to be something like

```php
$mock->shouldReceive('bootContainer')->with('-p "${:port}":7700 \
            -v "${:volume}":/data.ms \
            "${:organization}"/"${:image_name}":"${:tag}"', [
                "organization" => "getmeili",
                "image_name" => "meilisearch",
                "port" => 7700,
                "tag" => "v1.1.1",
                "volume" => null,
                "container_name" => "TO--meilisearch--v1.1.1",
            ])->once();
```

which doesn't look very nice, any suggestions?

Reply:
Eh, let's just keep it as you have it for now and I'll dig in later to see if we really need that. Thanks so much!